cli/main.rs

```diff
@@ -143,6 +143,10 @@ async fn run_subcommand(flags: Arc<Flags>) -> Result<i32, AnyError> {
       )
     }
     DenoSubcommand::Init(init_flags) => {
```

Comment:
It would be nice to keep the logic in here very simple and instead just have:

```rs
spawn_subcommand(async { tools::init::init_command(flags, init_flags).await })
```